### ISSUE TITLE
Phone (cannot hear/mic muted)

### ISSUE THREAD

**Issue by fusionneur** [sunilpaulmathew/De-Bloater#5]
I disabled these using de-bloater:
ok google enrollment
x google enrollment
google location history
photos
google pay
GOOGLE CONTACTS SYNC

Now, one of these, (not verysure which) BREAKS PHONE APP (cannot hear/mic muted). 

Now the problem with de-bloater is that, even if i restored all the above apps and rebooted, still couldn't call.
I had to remove the apk and magisk module + restart to make it work again.

Most probably it was due to the google location history being disabled. Then carrier location notification appeared after reboot.

**Comment by sunilpaulmathew** [comment]
De-Bloater really doesn't have any database to know the consequences of disabling an app. It is really difficult to do. The only thing possible is to use the app more carefully by the user. Just don't touch an app if you're not sure.

**Comment by vrealizebuildtools** [comment]
yes but it should restore to the initial state after restoring apk

**Comment by fusionneur** [comment]
yes but it should restore to the initial state after restoring apk

**Comment by sunilpaulmathew** [comment]
> yes but it should restore to the initial state after restoring apk

App will definitely restore the original APK file after a reboot, if you removed it from the Inactive applications page. However, the app settings removed will be lost forever. Also. the app which shows issues might be dependent on some other apps, that are still debloated. That's why, removing the module solves the issues on your side. The only option to solve this issue is to identify the culprit(s) and avoid them from debloating.

**Comment by fusionneur** [comment]
I found that the app that shows notification after disabling magisk module is com.oneplus.carrierlocation which i didn't even debloated. As you say , the cause of phone app stopping working might be due to this apk (which comes from oneplus) that has dependencies in some of the apks that i debloated. quite messy..
